test/src/e2e_vm_tests/test_programs/should_pass/forc/contract_dependencies/contract_a/Forc.lock

```diff
@@ -0,0 +1,16 @@
+[[package]]
+name = 'contract_a'
+source = 'root'
+contract_dependencies = [
```

Comment:
One more minor request I just noticed after addressing Bing's Q: Could we change this to `contract-dependencies` (use slash rather than underscore) to match the `Forc.toml` name? I think there's a `serde` attribute that allows for renaming a field to a kebab case name.